src/styles/variables.scss

```diff
@@ -17,4 +17,3 @@ $policy-color-str-2: #bdbdbd;
 $policy-color-str-3: #fff;
 $link-visited: #797979;
 $font-color-black: #000;
```

Comment:
It was good to declare the highlight color as a variable, maybe as `$highlight-color-yellow: #feff54;`